test/common/index.js

```diff
@@ -80,7 +80,12 @@ if (process.argv.length === 2 &&
           // If the binary is build without `intl` the inspect option is
           // invalid. The test itself should handle this case.
           (process.features.inspector || !flag.startsWith('--inspect'))) {
-        throw new Error(`Test has to be started with the flag: '${flag}'`);
+        const args = [...flags, ...process.execArgv, ...process.argv.slice(1)];
+        const options = { encoding: 'utf8', stdio: 'inherit' };
+        const result = spawnSync(process.execPath, args, options);
+        assert.ifError(result.error);
+        process.exit(result.status);
```

Comment:
If I understand correctly, this line is equivalent to `process.exit(0)`, because `result.error` will be set if `result.status !== 0`. If so, maybe an assert to that effect would be helpful to the reader: `assert.strictEqual(result.status, 0)`

Alternatively, you might consider making the parent process terminate the same way the child did:
```
if (result.signal) {
  process.kill(0, result.signal);
} else {
  process.exit(result.status);
}
```

The additionaly ifError() that is there now will cause (I  think) two stack traces for the same error if the sub-process (aka the actual test) fails. At least in my usage, that means the originating error might go off the top of my screen :-).

Reply:
@BridgeAR will you make this change, or are you avoiding working on this until there is more agreement the PR as a whole is going to get approved? I'd understand if the latter is the case :-), just wondering.

Reply:
Fixed. Thanks for the suggestion! I forgot about pid `0` and had to look up the man page for it. That's why I did not do it right away.